src/crypt/signOrder.ts

```diff
@@ -37,21 +36,21 @@ export const signOrder = async (
     baseAsset: baseAssetAddr,
     quoteAsset: quoteAssetAddr,
     matcherFeeAsset: serviceFeeAssetAddr,
-    amount: normalizeNumber(
+    amount: Number(normalizeNumber(
```

Comment:
Try to avoid conversion without checking data lose.
Consider using: https://gist.github.com/lobotomoe/1cdae384393311b2a2636ecf3bd87f3b